mod/wikirate_source/set/type/source/source_preview.rb

```diff
@@ -116,27 +152,24 @@
     }
   end
 
-  view :non_previewable ,:tags=>:unknown_ok do |args|
+  view :non_previewable ,tags: :unknown_ok do |_args|
     if file_card = Card[card.name+"+File"]
       %{<a href="#{file_card.attachment.url}" class="btn btn-primary" role="button">Download</a>}
     else
-      url_card = card.fetch(:trait=>:wikirate_link)
+      url_card = card.fetch(trait: :wikirate_link)
       url = url_card ? url_card.item_names.first : nil
       %{<a href="#{url}" class="btn btn-primary" role="button">Visit Original Source</a>}
     end
   end
 
-  view :preview_options, :tags=>:unknown_ok  do |args|
-
+  view :preview_options, tags: :unknown_ok  do |args|
     url = args[:url]
-    related_claim_wql = {:left=>{:type_id=>Card::ClaimID},:right=>"source",:link_to=>card.name,:return=>"count"}
-    related_metric_wql = {:type_id=>Card::MetricID, :right_plus=>[{:type_id=>Card::WikirateCompanyID}, :right_plus=>[{:type=>"year"}, :right_plus=>["source", {:link_to=>card.name}]]],:return=>"count"}
+    related_claim_wql = {left: {type_id: Card::ClaimID},right: "source",link_to: card.name,return: "count"}
```

Comment:
Line is too long. [107/80]<br>Space missing after comma.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.